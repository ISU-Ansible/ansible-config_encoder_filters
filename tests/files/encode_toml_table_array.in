array_arr1:
  - text: some text
    arr11:
      - 123
      - 987.654
  - bool1: yes
    bool2: "false"

array_arr2:
  products:
    - name: Hammer
      sku: 738594937
    - {}
    - name: Nail
      sku: 284758393
      color: gray

array_arr3:
  fruit:
    - name: apple
      physical:
        color: red
        shape: round
      variety:
        - name: red delicious
        - name: granny smith
    - name: banana
      variety:
        - name: plantain
