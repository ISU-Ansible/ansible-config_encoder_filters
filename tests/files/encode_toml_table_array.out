[[array_arr1]]
arr11 = [123, 987.654]
text = "some text"

[[array_arr1]]
bool1 = true
bool2 = "false"

[[products]]
name = "Hammer"
sku = 738594937

[[products]]

[[products]]
name = "Nail"
sku = 284758393
color = "gray"

[[array_arr3]]

  [[fruit]]
  name = "apple"

    ["fruit.physical"]
    color = "red"
    shape = "round"

    [["fruit.variety"]]
    name = "red delicious"

    [["fruit.variety"]]
    name = "granny smith"

  [[fruit]]
  name = "banana"

    [["fruit.variety"]]
    name = "plantain"
